lib/ma_crossover/events/data_managed_candles.js

```diff
@@ -56,45 +60,55 @@ const onDataManagedCandles = async (instance = {}, candles, meta) => {
         longIndicator.add(price)
       } else if (state.lastCandleLong.mts === lastCandle.mts) {
         longIndicator.update(price)
```

Comment:
do this:

1. create algo order
2. submit a few market orders, which will lead to candle updates
3. check csv, for every update to one entry a csv log entry is written:

```
long,Fri May 14 2021 18:24:00 GMT+0200 (Central European Summer Time),51162,close,EMA(6),51220.504795251836,0
short,Fri May 14 2021 18:24:00 GMT+0200 (Central European Summer Time),51162,close,EMA(3),51203.59108453605,0
long,Fri May 14 2021 18:24:00 GMT+0200 (Central European Summer Time),51162,close,EMA(6),51220.504795251836,0
short,Fri May 14 2021 18:24:00 GMT+0200 (Central European Summer Time),51162,close,EMA(3),51203.59108453605,0
```

is this intended @avsek477 - can you check with ivan?

Reply:
having updates in the log helps with debugging, as we have intermediate states there

Reply:
hmmmm

Reply:
yes, it's intended .. we log and calculate ma values for every candle we receive